tests/unit/conftest.py

```diff
@@ -116,14 +116,12 @@ def mock_airflow_connection_datarobot_jdbc(mocker, dr_jdbc_conn_details):
     mocker.patch.dict("os.environ", AIRFLOW_CONN_DATAROBOT_JDBC_DEFAULT=conn.get_uri())
 
 
-@pytest.fixture(autouse=True)
+@pytest.fixture()
 def mock_datarobot_basic_credentials(mocker, dr_basic_credentials_conn_details):
     credentials_create_mock = mocker.Mock(
         credential_id='test-credentials-id',
         name='datarobot_basic_credentials_test',
         credential_type='basic',
-        user=dr_basic_credentials_conn_details["login"],
-        password=dr_basic_credentials_conn_details["password"],
```

Comment:
Why is the username and password being deleted from the mock?

Reply:
its because its mock for Credential object which doesn't contain login/password

Reply:
Ah that's right, the secret data isn't actually contained in a DR API `Credential`.